xbmc/LangInfo.cpp

```diff
@@ -756,24 +755,23 @@ bool CLangInfo::SetLanguage(std::string language /* = "" */, bool reloadServices
       if (!addonMgr.GetAddon(language, addon, ADDON::AddonType::RESOURCE_LANGUAGE,
                              ADDON::OnlyEnabled::CHOICE_NO))
       {
-        CLog::Log(LOGFATAL, "CLangInfo::{}: could not find default language add-on '{}'", __func__,
-                  language);
+        CLog::LogF(LOGFATAL, "Could not find default language add-on '{}'", language);
         return false;
       }
     }
   }
 
-  CLog::Log(LOGINFO, "CLangInfo: loading {} language information...", language);
+  CLog::Log(LOGINFO, "Loading {} language information...", language);
```

Comment:
```suggestion
  CLog::LogF(LOGINFO, "Loading {} language information...", language);
```

Reply:
Nice, but something for another PR, as `CLog::Log` (other than `Clog::LogF` and `Clog::LogFC`) is not effected by the move from `__func__` to `std::source_location` in this PR.

Reply:
I agree, but then this PR should not touch these lines.

Reply:
Oh , Sorry, you are right. Will revert. Same for the other comment you made. 

What i had in mind is LOGINFO imo never should use hard core technical/debug information. Seeing a function name with LOGINFO feels wrong to me personally, but this is nothing for this PR.

Reply:
> I agree, but then this PR should not touch these lines.

Reverted the change.